src/Concordium.Sdk.csproj

```diff
@@ -45,29 +49,50 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="System.Linq.Async" Version="6.0.1" />
+    <PackageReference Include="WebAssembly" Version="1.2.1" />
   </ItemGroup>
 
   <ItemGroup>
-    <Protobuf Include="../concordium-grpc-api/v2/concordium/types.proto" GrpcServices="Client">
+    <Protobuf Include="../concordium-base/concordium-grpc-api/v2/concordium/types.proto" GrpcServices="Client">
       <Access>Public</Access>
       <ProtoCompile>True</ProtoCompile>
       <CompileOutputs>True</CompileOutputs>
       <OutputDir>obj\Debug\net6.0\</OutputDir>
       <Generator>MSBuild:Compile</Generator>
       <Link>v2/concordium/types.proto</Link>
-      <ProtoRoot>../concordium-grpc-api</ProtoRoot>
+      <ProtoRoot>../concordium-base/concordium-grpc-api</ProtoRoot>
     </Protobuf>
-    <Protobuf Include="../concordium-grpc-api/v2/concordium/service.proto" GrpcServices="Client">
+    <Protobuf Include="../concordium-base/concordium-grpc-api/v2/concordium/service.proto" GrpcServices="Client">
       <Access>Public</Access>
       <ProtoCompile>True</ProtoCompile>
       <CompileOutputs>True</CompileOutputs>
       <OutputDir>obj\Debug\net6.0\</OutputDir>
       <Generator>MSBuild:Compile</Generator>
       <Link>v2/concordium/services.proto</Link>
-      <ProtoRoot>../concordium-grpc-api</ProtoRoot>
+      <ProtoRoot>../concordium-base/concordium-grpc-api</ProtoRoot>
     </Protobuf>
   </ItemGroup>
 
+
+  <ItemGroup>
+    <!-- Include librust.dll on Windows -->
+    <None Include="../rust-bindings/target/release/librust_bindings.dll" Condition="'$(IsWindows)'=='true'">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <!-- Include librust.dylib on macOS -->
+    <None Include="../rust-bindings/target/release/librust_bindings.dylib" Condition="'$(IsOSX)'=='true'">
```

Comment:
Is this x86 or aarch64? Will it depend on what machine this thing is built?

Reply:
Yes dependent on which platform it is build on the `librust_bindings.*` will be different.

Please note that this is required for local execution. The packaging, which includes all binaries for Linux, iOS, and Windows, will be handled in another PR.

Regarding your query about 'Is this x86 or aarch64,' could you provide further clarification or context? I'm unsure about the specific information you're seeking.

Reply:
There are currently two mac platforms widely used, the older x86-64 intel macs and the newer M{1,2,3} based ones. You can use the x86 code on m1 macs via rosetta but not the other way around.

So my concern is that we package the SDK in the right way so that it's actually usable on both types of platform.

Reply:
In this PR, which handles packaging, I have made a pipeline which builds on mac os. 
https://github.com/Concordium/concordium-net-sdk/pull/79

Looking at the mac os build step I can see the version used for rust is `1.68-x86_64-apple-darwin`. So from what I understand we are packaging correctly.
https://github.com/Concordium/concordium-net-sdk/actions/runs/6971280381/job/18970984682

I've followed the flow used in the Java SDK release pipeline. If we haven’t encountered any issues related to the Java release I wouldn’t expect any on the .NET.
https://github.com/Concordium/concordium-java-sdk/blob/main/.github/workflows/release.yml#L114

Reply:
The Java flow hasn't really been tested, it's very new.

But it does look that macos-latest is still intel based so it is fine.

Thanks for checking up.